BPR.Analysis/Services/AnalysisService.cs

```diff
@@ -128,4 +148,65 @@ internal static List<Violation> GetNamespaceAnalysis(List<NamespaceDirective> na
 
         return violations;
     }
+
+    private async Task<List<NamespaceDirective>> GetNamespaceDirectivesAsync(string folderPath)
+    {
+        List<NamespaceDirective> matches = new();
+        var files = Directory.GetFiles(folderPath, "*", SearchOption.AllDirectories);
+        foreach (var file in files)
+        {
+            if (file.EndsWith(".cs") || file.EndsWith(".cshtml"))
+            {
+                var content = await File.ReadAllLinesAsync(file, Encoding.UTF8);
```

Comment:
I see some similar code with GetUsingDirectivesAsync, maybe a method can help to diminish code duplication

Reply:
Can't see an easy way to do so. better in another PR as I touched way too many here